File: modified_scripts/scala/is_palindrome/deepseek-coder_1.3b.scala

def is_palindrome(x : Int) : Boolean = {  
    var y : Int = x  
    var rev : Int = 0 
    while(y > 1) { 
        rev = ((rev * 10) + y % 10); 
        y = y / 10;
    }  
     rev;  
}

File: modified_scripts/julia/is_palindrome/deepseek-coder_1.3b.jl

function is_palindrome(s)
    length = div(length(s),2 ) 
    
    i = 1           
                     
    j = length       
    
    
    t = ""             
    
    while i <= j
        if s[i] != s[j]   
            return false    
        else                
            i += 1          
            j -= 1          
        end               
     end                
